entrypoint.py

```diff
@@ -171,6 +218,7 @@ async def rpc_handler(request):
         web.get("/api/ls", ls_handler),
         web.post("/api/ls", ls_handler),
         web.post("/api/pub", pub_handler),
+        web.get("/api/rest/pub", pub_rest_handler),
```

Comment:
Shouldn't this be a post and shouldn't /api/pub be switched to a get?

Reply:
Yep, I swapped these around while we were slacking last night. Bad partial edit. Fixed.